src/odoo_xmlrpc_wrapper/odoo_xmlrpc_wrapper.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         if self.successful:
             print(self.status)
 
-    def satus(self) -> str:
+    def status(self) -> str:
```

Comment:
Hello ! I think you'll also need to add the `@property` to the method (or call `self.status()`) ! Thanks